images/kubevirt-rootfs.yml.in

```diff
@@ -0,0 +1,99 @@
+kernel:
```

Comment:
Can't you use the yq way to patch in the differences instead of duplicating everything? See other yq files in this directory.

Reply:
Yeah I am know its not a clean way to do and I am looking for such tools to patch the differences. Thanks for pointing it out, I will look at yq and make changes.